### ISSUE TITLE
html/select_element Screen Reader issues with collapsed select elements

### ISSUE THREAD

**Issue by steveja11y** [accessibilitysupported/a11ysupport.io#250]
Hi! Just wanted to bring attention to some quirky screen reader behavior and didn’t see any info about it on the main `<select>` page. Hopefully this info is helpful for documenting support or creating a test case around this scenario.
 

**Issue Summary**
When using a screen reader with a `<select>` element in focus and in the collapsed state, users can change the selection using letter keys on the keyboard without triggering the component’s expanded state. Consequently, while these changes are occurring and visible to sighted users, screen reader users will receive no announcements that the value of the `<select>` has changed.
[Example video of this bug in action.](https://youtu.be/Vg_B-eQryVA)
 
I personally tested this out with the following combo:
 

- OS: macOS v12.6
- Browser: Safari v16.1 (17614.2.9.1.13, 17614)
- AT: VoiceOver

**Testing Approach**
- Site: https://a11ysupport.io/tests/html/html/select.html
- Setup: Desktop browser and screen reader 
- Instructions: 
     - Using the tab key, move focus to the `<select>` element.
     - With the `<select> `in focus and in the collapsed state, use letter keys (I.e. a,b,c) to update the selected option.
- Expected Behavior:
     - The screen reader will announce changes to the `<select>` element as different letter keys match with various list options.
- Actual Behavior:
     - No screen reader announcements are provided for collapsed `<select>` elements.

**WCAG Implications**
WCAG 2.0 S.C. 4.1.2: Name, Role, Value (Level A)
This error seems to violate WCAG success criterion 4.1.2: Name, Role, Value.
"For all user interface components (including but not limited to: _form elements_, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and _values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies."

**Related Bug Reports:**
It seems as though this error originates from the browser level and numerous major browsers have open/unresolved tickets related to this issue:

- [Apple Safari (Webkit)](https://bugs.webkit.org/show_bug.cgi?id=209076)
- [Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=759369&q=screen%20reader%20accessibility%20component%3ABlink%3EForms%3ESelect&can=1)
- [Microsoft Edge](https://github.com/microsoft/fluentui/issues/5752)
- [ Mozilla Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=126379)

Finally, there is also an unresolved HTML spec bug report related to this `<select>` behavior.
- [HTML Spec Bug Report ](https://github.com/whatwg/html/issues/3063)

**Comment by mfairchild365** [comment]
Thank you for the report! I’ll see if I can re-test this.

**Comment by mfairchild365** [comment]
fixed in https://github.com/accessibilitysupported/a11ysupport.io/commit/11eb11165784ff4c68de10c72fb48cb6a7477ee2

Note: I could only duplicate this bug in VO+Safari on MacOS. All other desktop screen reader / browser combinations worked as expected.
